sktime/forecasting/model_selection/_tune.py

```diff
@@ -336,6 +342,9 @@ class ForecastingGridSearchCV(BaseGridSearch):
     error_score: numeric value or the str 'raise', optional (default=np.nan)
         The test score returned when a forecaster fails to be fitted.
     return_train_score: bool, optional (default=False)
+    backend: str, optional (default=None)
+        Specify the parallelisation backend implementation in joblib, where
+        'loky' is used by default.
```

Comment:
confusing - is it None or `"loky"`? Suggest: to avoid confusion, use the "effective" default which is `"loky"`.